src/components/Upgrade/content.js

```diff
@@ -0,0 +1,116 @@
+import large1 from './highlights/large/1.png'
+import large2 from './highlights/large/2.png'
+import large3 from './highlights/large/3.png'
+import large4 from './highlights/large/4.png'
+import large5 from './highlights/large/5.png'
+
+import small1 from './highlights/small/1.png'
+import small2 from './highlights/small/2.png'
+import small3 from './highlights/small/3.png'
+import small4 from './highlights/small/4.png'
+import small5 from './highlights/small/5.png'
+
+export const banner = {
+  text: {
+    large: 'Upgrade your organization to the 0.7 Bella release! 🎉',
+    small: 'Upgrade to 0.7 Bella! 🎉',
+  },
+  button: 'More info',
+}
+
+export const highlights = [
```

Comment:
I think this now lives in `./highlights.js` (or that file could be removed)?

Reply:
Yes, `highlights.js` shouldn’t have been pushed :s